sql/moz-fx-data-shared-prod/search_terms_derived/sanitation_job_metadata/init.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS
+  `moz-fx-data-shared-prod.search_terms_derived.sanitation_job_metadata`
+AS
+SELECT
+  *
+FROM
+  `mozdata.search_terms_unsanitized_analysis.prototype_sanitation_job_metadata`
```

Comment:
I think we probably want to use the productionization process to to create a `schema.yaml` file for this table, but I've asked @jklukas to opine.

Reply:
Current best practice is to have a `schema.yaml` and a `bigquery` section in `metadata.yaml` that defines partitioning, etc. In which case you don't need an `init.sql`